test/jasmine/tests/pie_test.js

```diff
@@ -972,7 +972,7 @@ describe('Pie traces', function() {
             textposition: [['outside', 'outside', 'inside', 'inside', 'outside']],
             'outsidetextfont.size': 12
         }, {
-            t: '~=', l: '~=',
+            t: '~=', l: 'shrunk',
```

Comment:
Bring back the original test which was tweaked in https://github.com/plotly/plotly.js/pull/5225/commits/e01d09ab0c22c0d4c30b0a3146079bda3c0e1a58.